qa/tasks/scripts/munge_policy.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+# munge_policy.sh
+#
+# Adds or removes roles from policy.cfg
+#
+
+set -ex
+
+OPERATION=$1 # add_node_as_role / remove_storage_only_node
```

Comment:
I guess it is "action", not an "operation".

Reply:
I guess it should be same style naming, like "add_node", "del_node" to be corresponding to description "Adds or removes roles from policy.cfg".